qa/pull-tester/rpc-tests.py

```diff
@@ -266,7 +268,23 @@ def main():
         subprocess.check_call((config["environment"]["SRCDIR"] + '/qa/rpc-tests/' + test_list[0]).split() + ['-h'])
         sys.exit(0)
 
-    run_tests(test_list, config["environment"]["SRCDIR"], config["environment"]["BUILDDIR"], config["environment"]["EXEEXT"], args.jobs, args.coverage, passon_args)
+    if (args.rpcgroup == -1) != (args.machines == -1):
+        print("ERROR: Please use both -m and -r options when using parallel rpc_groups.")
+        sys.exit(0)
+    if args.rpcgroup >= args.machines:
+        print("ERROR: -r/--rpcgroup must be less than -m/--machines")
+        sys.exit(0)
+    if args.rpcgroup != -1 and args.machines != -1:
+        k, m = divmod(len(test_list), args.machines)
+        k = k if m == 0 else k+1
+        # Ceiling division using floor division, by inverting the world.
+        # https://stackoverflow.com/a/17511341
+        k -(len(test_list) // -args.machines)
```

Comment:
Whoops, my original suggestion was missing an `=`:
```suggestion
        # Ceiling division using floor division, by inverting the world.
        # https://stackoverflow.com/a/17511341
        k = -(len(test_list) // -args.machines)
```